webapp/channels/src/actions/status_actions.ts

```diff
@@ -16,9 +16,10 @@ import {loadCustomEmojisForCustomStatusesByUserIds} from 'actions/emoji_actions'
 import type {GlobalState} from 'types/store';
 
 export function loadStatusesForChannelAndSidebar(): ActionFunc<boolean, GlobalState> {
+    const userIds = new Set<string>();
```

Comment:
This can be moved into the inner function. It doesn't really affect how long these variables last to have them out here. Thunk actions aren't like `makeMapStateToProps` where `connect` reuses the returned function multiple times so that anything defined in the outer function can also be reused

Reply:
moved inwards